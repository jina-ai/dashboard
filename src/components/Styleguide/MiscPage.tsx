/** @jsxRuntime classic */
/** @jsx jsx */
import { css, jsx, useTheme } from "@emotion/react"
import { PageTitle } from "../Common/PageTitle"
import SpinningLoader from "../Common/SpinningLoader"

const MiscPage = () => {
  const { palette } = useTheme()
  return (
    <div>
      <PageTitle title={"Misc"} className={""} />
      <span
        css={css`
          color: ${palette.headerTextColor};
          padding: 8px;
        `}
      >
        Loading Spinner
      </span>
      <SpinningLoader />
    </div>
  )
}

export default MiscPage
