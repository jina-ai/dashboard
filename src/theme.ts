const PERSIAN_GREEN = "#009999"
const GOLDEN_TANOI = "#FFCC66"
const BITTER_SWEET = "#ff6666"
const SCOOTER_GREEN = "#32C8CD"
const WHITE = "#FFFFFF"
const DADGREY = "#DAD7FE"
const SKY_DIVING_IN_THE_BLUE = "#007AFF"
const QUICK_SILVER = "#A6A6A6"
const HONEYBEE = "#FFCC66"
const BLACK = "#000000"

const theme = {
  palette: {
    background: WHITE,
    headerTextColor: BLACK,
    primary: PERSIAN_GREEN,
    secondary: GOLDEN_TANOI,
    warning: GOLDEN_TANOI,
    error: BITTER_SWEET,
    success: SCOOTER_GREEN,
    info: PERSIAN_GREEN,
    critical: "#ff4540",
    debug: "#7E7278",
    tagBackground: DADGREY,
    tagPlatformBackground: HONEYBEE,
    mutedText: QUICK_SILVER,
    highlight: SKY_DIVING_IN_THE_BLUE,
  },
}

export { theme }
