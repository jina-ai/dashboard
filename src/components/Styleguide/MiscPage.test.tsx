import React from "react"
import renderer from "react-test-renderer"
import { ThemeProvider } from "@emotion/react"
import { theme } from "../../theme"
import MiscPage from "./MiscPage"

test("Misc page", () => {
  const component = renderer.create(
    <ThemeProvider theme={theme}>
      <MiscPage />
    </ThemeProvider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
