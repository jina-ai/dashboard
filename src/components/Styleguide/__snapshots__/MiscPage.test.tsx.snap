// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Misc page 1`] = `
<div>
  <div
    className="text-center text-md-left mb-sm-0 col-sm-3 col-12"
  >
    <h3
      className="page-title"
      data-name="title"
    >
      Misc
    </h3>
    <p
      className="page-subtitle mt-2 mb-0"
    />
  </div>
  <span
    className="css-104iut4-MiscPage"
  >
    Loading Spinner
  </span>
  <div
    className="css-18vjixk"
  >
    <div
      className="css-11ue89k"
    >
      <div
        className="css-jn4y13"
      />
    </div>
  </div>
</div>
`;
